src/main/java/lotto/Controller.java

```diff
@@ -0,0 +1,67 @@
+package lotto;
+
+import lotto.domain.Computer;
+import lotto.domain.LottoRank;
+import lotto.domain.LottoResult;
+import lotto.domain.lottonumber.WinningLotto;
+import lotto.views.BuyLottoView;
+import lotto.views.LottoResultView;
+import lotto.views.WinningLottoView;
+
+import java.util.List;
+
+public final class Controller {
+    public static final int PRICE_OF_LOTTO = 1000;
+    private static List<List<Integer>> lotto;
+    private static WinningLotto winningLotto;
+
+    public void startLottoGame() {
+        try {
+            buyLotto();
+            getWinningLotto();
+            getLottoResult();
+        } catch (IllegalArgumentException e) {
+            e.printStackTrace();
+            return;
+        }
+    }
+
+    private void buyLotto() {
+        BuyLottoView buyLottoView = new BuyLottoView();
+        try {
+            lotto = new Computer().buyLottoByPrice(buyLottoView.readPrice());
+            buyLottoView.getViewPurchasedLotto(lotto);
+        } catch (IllegalArgumentException e) {
+            System.out.println("[ERROR] 올바르지 않은 입력");
+        }
+    }
+
+    private void getWinningLotto() {
+        WinningLottoView winningLottoView = new WinningLottoView();
+        winningLotto = new WinningLotto(winningLottoView.readWinningLottoNumbers(), winningLottoView.readBonusNumber());
+    }
+
+    private void getLottoResult() {
+        LottoResult lottoResult = new LottoResult(lotto, winningLotto);
+        LottoResultView lottoResultView = new LottoResultView();
+
+        for (LottoRank rank : LottoRank.values()) {
+            int amount = lottoResult.getNumberOfRanks(rank);
+            if (rank == LottoRank.UNRANKED) continue;
+
+            printResult(lottoResultView, rank, amount);
+        }
+        printAverageProfit(lottoResult, lottoResultView);
+    }
+
+    private void printAverageProfit(LottoResult lottoResult, LottoResultView lottoResultView) {
+        double average = lottoResult.getAverageProfit(lotto.size() * PRICE_OF_LOTTO);
+        System.out.println(lottoResultView.getViewAverageProfit(average));
```

Comment:
출력을 하는 것까지 View의 역할이 아닐까요?
```suggestion
    lottoResultView.printAverageProfit(average);
```

Reply:
```suggestion
        lottoResultView.printViewAverageProfit(average);
    }
``` 



## LottoResultView.java
``` 
  public void printViewAverageProfit(double average) {
          System.out.printf("총 수익률은 %.1f%%입니다.\n", average * 100);
      }
```
view에서 출력하도록 클래스를 옮겼습니다